create-release.html.md.erb

```diff
@@ -131,7 +131,7 @@ case $1 in
 
   start)
     mkdir -p $RUN_DIR $LOG_DIR
-    chown -R vcap:vcap $RUN_DIR $LOG_DIR
+    chown vcap:vcap $RUN_DIR $LOG_DIR
```

Comment:
Doing -R in the startup script has some side-effects that (probably) most users don't think about when copying this from bosh.io. So while some users might actually want that (or a similar) side-effect, I'd say let's optimize for the standard user. Wdyt?